scripts/fuzz_opt.py

```diff
@@ -311,6 +311,8 @@ def is_git_repo():
     'exception-handling.wast',
     'translate-eh-old-to-new.wast',
     'rse-eh.wast',
+    # Non-UTF8 strings trap in V8
```

Comment:
Trap or validation failure? Should we make them validation failures in Binaryen as well?

Reply:
Hmm, I think it was a trap actually but I'd need to find the testcase again to check.

But I don't think these should fail to validate - non-utf8 strings are ok in the stringref proposal, as you can use other encodings there. It should only trap if you use the utf8 encoding I think.

Reply:
According to the overivew, the string literals must be valid WTF-8, although it doesn't say what happens (presumably a decoding error) if they're not: https://github.com/WebAssembly/stringref/blob/main/proposals/stringref/Overview.md#string-literal-section

Reply:
Oh, good catch... but then why does the proposal also support other encodings elsewhere?

Reply:
Presumably they want to support encoding and decoding to all kinds of formats for interoperability with other systems, but there's no need to support more than one format for literals because they can be encoded correctly, whatever correctly means, during compilation.